README.md

```diff
@@ -193,6 +193,10 @@ Tracks an existing change from the current branch
 
 You will need JDK 9+.
 
+Currently with OpenJDK 11 there is a need for a newer version of gradle, which causes issues with the buildscript, and with OpenJDK 10 I came across a certificate issue when the buildscript tried to download the Azul JRE 9 to build the binary.
```

Comment:
Please remove this part.
You can instead say that it needs exactly JDK 9.

Please know that the certificate issue comes from the fact that JDK 10 cacerts must be obsolete.